Formula/b/btop.rb

```diff
@@ -17,19 +17,29 @@ class Btop < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "af5c08bf52c9d84b54b3fe3992240a50c048ff3b897e8ee13f428fac4db76e0f"
   end
 
+  depends_on "coreutils" => :build
+
+  # Requires Clang 16.0.0, GCC 10.1.0
   on_macos do
-    depends_on "coreutils" => :build
```

Comment:
Is there a reason we need a newer `coreutils` on Linux now? Is it using something that is not preinstalled on GNU/Linux distros?